api/channel.go

```diff
@@ -21,6 +21,7 @@ func InitChannel() {
 	BaseRoutes.Channels.Handle("/", ApiUserRequired(getChannels)).Methods("GET")
 	BaseRoutes.Channels.Handle("/more", ApiUserRequired(getMoreChannels)).Methods("GET")
 	BaseRoutes.Channels.Handle("/counts", ApiUserRequired(getChannelCounts)).Methods("GET")
+	BaseRoutes.Channels.Handle("/unread", ApiUserRequired(getChannelsUnread)).Methods("GET")
```

Comment:
Why call this `/unread` and `getChannelsUnread`? It's more like `getChannelMembers`.